packages/inventory/src/constants.ts

```diff
@@ -8,6 +8,8 @@ export const LOCATIONS_GET_ALL_SYNC_ENDPOINT = 'location/getAll';
 export const INVENTORY_SERVICE_POINT_LIST_VIEW = '/inventory/list';
 export const INVENTORY_SERVICE_POINT_PROFILE_VIEW = '/inventory/profile';
 export const INVENTORY_BULK_UPLOAD_URL = '/inventory/upload';
+export const INVENTORY_ADD_SERVICE_POINT = '/inventory/servicePoint/add';
+export const INVENTORY_EDIT_SERVICE_POINT = '/inventory/servicePoint/edit';
```

Comment:
In future implementations could we pass the URLs as a props from the client instead of package to client?

Reply:
We can make them configurable, such as these can be the defaults

Reply:
yeah sure, we could create a dicussion issue and meet for this, to decide on how best to handle this. #419